extensions/escape-game/standalone.mjs

```diff
@@ -112,6 +113,27 @@ class Standalone {
       v.registerExtStateFunc('showTerminal', 'escape-game', 'sf_showTerminal');
       v.registerExtStateFunc('checkIsInFinalizedTeam', 'escape-game', 'sf_checkIsInFinalizedTeam');
     });
+    this.doorCells = [{'x': 9, 'y': 12, 'h': 1, 'w': 1}];
```

Comment:
If you intend to have only 1 door, then you can store this in the config, see example below. i.e.
```
this.doorCells = [config.get('escape-game.doorPosition')];
```
Otherwise, I would recommend storing the door location as a cellset (seems a little bit elaborate, not sure if we need this.